core/server/apps/amp/tests/amp_content_spec.js

```diff
@@ -119,11 +121,11 @@ describe('{{amp_content}} helper', function () {
 
         it('can transform img tags to amp-img', function (done) {
             var testData = {
-                    html: '<img src="/content/images/ghost.png" alt="The Ghost Logo" />',
+                    html: '<img src="/content/images/2016/08/scheduled2-1.jpg" alt="The Ghost Logo" />',
```

Comment:
I changed this to test with real existing images, but this is extremely ugly and bad. I guess, the tests since the latest update to Amperize worked before only by coincidence? Since this update, Amperize will return errors instead of default values.

I really need to stub Amperize, to make this ugly stuff go away. So far I couldn't make a stub work. Would appreciate some help here... @ErisDS, @kirrg001